src/components/map/MapService.js

```diff
@@ -1292,11 +1290,12 @@ goog.require('ga_urlutils_service');
           }
           var defer = $q.defer();
           var view = map.getView();
-          var source = view.getCenter();
-          var dist = Math.sqrt(Math.pow(source[0] - dest[0], 2),
-              Math.pow(source[1] - dest[1], 2));
-          var duration = Math.min(Math.sqrt(300 + dist /
-              view.getResolution() * 1000), 3000);
+          // TODO: use or not the duration?
+          // var source = view.getCenter();
+          // var dist = Math.sqrt(Math.pow(source[0] - dest[0], 2),
+          //    Math.pow(source[1] - dest[1], 2));
+          // var duration = Math.min(Math.sqrt(300 + dist /
+          //    view.getResolution() * 1000), 3000);
```

Comment:
I was suprised to have a duration of 0 in the animate function, so the code commented was useless. I can remove it.